taxonomies/categories.txt

```diff
@@ -108410,6 +108461,7 @@ ciqual_food_name:fr:Blé germé, cru
 en:Alfalfa sprouts
 fr:Pousses d'alfalfa
 en:Germinados de alfalfa
+pl:Kiełki lucerny
```

Comment:
This one is already associated to en:fresh alfalfa sprouts, could you rename one of the two?

ERROR - pl:kiełki-lucerny already is associated to en:fresh-alfalfa-sprouts (categories) - pl:kiełki-lucerny cannot be mapped to entry en:alfalfa-sprouts